app/config/events.php

```diff
@@ -67,4 +79,22 @@
     'users.sessions.delete' => [
         'description' => 'This event triggers when a user session is deleted from users API.',
     ],
+    'teams.create' => [
+        'description' => 'This event triggers when a team is created.',
+    ],
+    'teams.update' => [
+        'description' => 'This event triggers when a team is updated.',
+    ],
+    'teams.delete' => [
+        'description' => 'This event triggers when a team is deleted.',
+    ],
+    'teams.memberships.create' => [
+        'description' => 'This event triggers when a team memberships is created.',
+    ],
+    'teams.memberships.update.status' => [
```

Comment:
Can this not be 

```suggestion
    'teams.memberships.update' => [
```

Reply:
Might need to re-think the naming conventions of the events name, might require extra work for migration from old setup.